test/fixtures/vcr/relay_group_add_relay.json

```diff
@@ -11,14 +11,14 @@
       "url": "http://localhost:4000/v1/token"
     },
     "response": {
-      "body": "{\"token\":{\"value\":\"w05n5iZL4peQoZ1gibP1MR1TgvSTXrIwZzCIJYRjR+0=\"}}",
```

Comment:
What do you mean? VCR forces you to rerecord the cassette if you change the test. Or am I misunderstanding?